test/1_stdlib/Float.swift

```diff
@@ -222,13 +222,24 @@ func checkQNaN(_ qnan: TestFloat) {
   _precondition(qnan.floatingPointClass == .quietNaN)
 }
 
+func checkSNaN(_ snan: TestFloat) {
+  checkNaN(snan)
+// FIXME: i386 doesn't fully support signaling NaN.
```

Comment:
The fractal complexity of x86 never ceases to amaze. If we're willing to assume SSE2, returning in xmm registers on i386 would be a nice improvement…